windows/check_quiesce_snapshot/prepare_quiesce_snapshot.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Set fact of the remote path in guest OS"
+  ansible.builtin.set_fact:
+    quiesce_scripts_freeze_thaw_path: "C:\\Windows\\"
+    quiesce_scripts_tools_path: "C:\\Program Files\\VMware\\VMware Tools\\backupScripts.d\\"
+    quiesce_log_file_path: "C:\\vmbackup.log"
+    quiesce_snapshot_name: "quiesce_{{ current_test_timestamp }}"
+
+# Get existing *vss_manifests*.zip files in VM folder
+- name: "Get all VM files in VM folder"
+  include_tasks: ../../common/vm_get_config.yml
+  vars:
+    property_list:
+      - 'layoutEx.file'
```

Comment:
May I know whey here replaced `hw_files` with `layoutEx.file`? The former can directly get the file list, but the latter need a map filter to get them.

Reply:
Since not all the files in the VM folder are listed in the `hw_files`, here need '.zip' file created when taking quiesce snapshot.